cluster/src/main/java/com/hartwig/pipeline/calling/sage/SageCommandBuilder.java

```diff
@@ -68,6 +69,11 @@ public SageCommandBuilder addCoverage() {
         return this;
     }
 
+    public SageCommandBuilder addShallowSomaticMode() {
+        this.shallowSomaticMode = true;
```

Comment:
perhaps we should enforce that somatic mode is enabled as well? otherwise you might end up calling this method, thinking your in shallow and somatic mode.

Reply:
Added state checking in the build function plus tests for the more plausible errors